.circleci/config.yml

```diff
@@ -33,6 +33,9 @@ aliases:
   - &openjdk8-scala2_12_environment
     docker:
       - image: circleci/openjdk:8-jdk
+        environment:
+          # https://circleci.com/docs/2.0/java-oom/
+          _JAVA_OPTIONS: "-Xms128m -Xmx2g"
```

Comment:
@jisantuc an attempt to avoid accident OOMs; `2g` is enough even for GT tests; if that won't work - probably it is a sign that smth is not good with tests.